controllers/component_build_controller_test.go

```diff
@@ -1349,6 +1522,8 @@ var _ = Describe("Component initial build controller", func() {
 			deleteComponent(resourceKey)
 			deleteComponentPipelineRuns(resourceKey)
 			deleteSecret(pacSecretKey)
+			// wait for pruner operator to finish, so it won't prune runs from new test
+			time.Sleep(time.Second)
```

Comment:
Have you got an idea why it works without sleep for the whole file?

Reply:
Could one component per test be helpful? Or, launch the pruner controller only before tests that test pruner?

Reply:
it would help for sure, but I am not going to change now everything

Reply:
> it would help for sure, but I am not going to change now everything

Yeah, this can be done separately.